### ISSUE TITLE
Question about Linear probing training

### ISSUE THREAD

**Issue by longtaojiang** [SilentView/GigaTok#10]
Thank you for completing such an excellent work. 
When I tried to reproduce linear probing acc, I found that its convergence was very slow. I am not sure if this is normal, after all, intuitively speaking, this is just a linear layer with a batch norm.
May I ask about your learning rate, batch size, GPU count, and epochs on the ImageNet training set when training linear probing to obtain the results in Table 3?

**Comment by SilentView** [comment]
Thank you for your interest. 

About linear probing experiments for gpt models, we follow the settings of MAE: https://github.com/facebookresearch/mae. 
In our repo, you can check how to run `train_lm_lin_probe_and_eval.sh` in https://github.com/SilentView/GigaTok/blob/master/Detailed_instructions.md#for-autoregressive-models.

and here you can find the input args::
https://github.com/SilentView/GigaTok/blob/c4aafa6d931faa83b2c50676e30a109f19b45d98/scripts/composite_cmd/train_lm_lin_probe_and_eval.sh#L98

More specifically, you can see the base learning rate, epochs, and batch size below. The learning rate will be automatically adjusted with different batch sizes and the base learning rate.
```shell
bash scripts/train_lin_probe_gpt.sh \
    --save-path ${PROJECT_ROOT}/results/lin_probe_gpt/ \
    --data-root ${IMGNET_ROOT} \
    --image-size 256 \
    --model-config ${TOK_CONFIG} \
    --gpt-model ${GPT_MODEL} \
    --epochs 90 \
    --blr 0.1 \
    --sub-exp-dir ${LM_LIN_EXP_DIR} \
    --batch-size $LIN_BSZ \
    --mixed-precision ${LM_PRECISION} \
    --num-code ${NUM_CODE} \
    --vq-ckpt $VQ_CKPT \
    --ckpt-epoch ${CKPT_EPOCH} \
    --gpt-ckpt ${GPT_CKPT} \
    $QK_NORM_FLAG
```

We typically train linear probing on 32 A800 or 64 V100 GPUs, with batch size 128. This can be kind of costly, but we just strictly follow the approach of MAE. The final training loss is around 0.4764 for the LlamaGen-XXL model in Table 3.

**Comment by longtaojiang** [comment]
Thank you for your quick reply!
